### PR TITLE
fixing IE issues

### PR DESCRIPTION
**Pull request by Guntrisoft**
- added width/height to ma says svg to ensure it renders correctly
- added background-size aswell to ensure ma says renders correctly
- create a fallback font family for museo_slab
- re done the inline callout stripy background, and make sure it
  works across browsers
- ensure the height of the social buttons is specified
- ensure title on homepage is centered
- re-worked tiles on homepage to ensure they work across browsers

### CODE REVIEW

**Comment by Guntrisoft** [comment]
@benbarnett

File: app/assets/stylesheets/components/article_body/_inline_callout.scss [hunk 1]
```diff
@@ -4,8 +4,8 @@
   display: block;
   overflow: hidden;
   position: relative;
-  background: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0" y="0" viewBox="0 0 100 100" enable-background="new 0 0 100 100" xml:space="preserve"><line fill="none" stroke="%23303030" stroke-miterlimit="10" x1="100" y1="0" x2="0" y2="100"/></svg>');
-  background-size: 25px;
+  background: url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjxzdmcgdmVyc2lvbj0iMS4xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iNDAiIGhlaWdodD0iNTAiIHg9IjBweCIgeT0iMHB4IiB2aWV3Qm94PSIwIDAgNDAgNTAiIHhtbDpzcGFjZT0icHJlc2VydmUiPjxsaW5lIHN0cm9rZT0iI2NlY2VjZSIgc3Ryb2tlLXdpZHRoPSIyIiB4MT0iMCIgeTE9IjUxIiB4Mj0iNDAiIHkyPSItMSIgLz48L3N2Zz4NCg==');
+  background-size: 20px 26px;
```
Comment (benbarnett) [inline app/assets/stylesheets/components/article_body/_inline_callout.scss]: Is this still needed now we have the size set in the SVG?

**Comment by darokel** [comment]
:fire:

File: app/assets/stylesheets/components/ma-says/_ma-says.scss [hunk 2]
```diff
@@ -8,6 +8,7 @@
   background: asset-url('ma-says.png') 0 0 no-repeat;
   background-image: linear-gradient(transparent, transparent), asset-url('ma-says.svg');
   background-repeat: no-repeat;
+  background-size: 193px 75px;
```
Comment (benbarnett) [inline app/assets/stylesheets/components/ma-says/_ma-says.scss]: And here

File: app/assets/stylesheets/components/article_body/_inline_callout.scss (hunk 1 above)
Comment (Guntrisoft) [inline app/assets/stylesheets/components/article_body/_inline_callout.scss]: short answer is yes - removing the background-size changes the outcome on Chrome :)

File: app/assets/stylesheets/components/ma-says/_ma-says.scss (hunk 2 above)
Comment (Guntrisoft) [inline app/assets/stylesheets/components/ma-says/_ma-says.scss]: This is needed too, as i've realised that it needs to be re-sized down on the homepage where there is a ma says link in the campaign section.

**Pull request merged**